ci/pipeline.yml

```diff
@@ -436,6 +436,31 @@ jobs:
           run:
             path: pipeline-tasks/ci/tasks/open-bump-proto-pr.sh
 
+  - name: test-android
+    plan:
+      - in_parallel:
+          - { get: galoy-mobile-repo, passed: "release", trigger: true }
```

Comment:
@bodymindarts Is this the correct spot to trigger the mobile e2e tests from when "release" passes?